audio_sync/cli.py

```diff
@@ -126,6 +135,36 @@ def _PlotResults(
   print(output)
 
 
+def _PlotAsciiGraph(
+    latencies, start_time, dots_per_usec=0.01, latency_threshold_secs=0.001):
+  """Plots all latencies with timestamp in an ASCII timeline."""
+  thresh_in_dots = int(latency_threshold_secs * SECS_IN_MSEC * dots_per_usec)
+  for latency in latencies:
+    if math.isnan(latency[1]):
+      usecs = 9999.0
+    else:
+      usecs = SECS_IN_MSEC * latency[1]
+    dots_total = int(abs(usecs) * dots_per_usec)
+    dots_below_thresh = min(dots_total, thresh_in_dots)
+    filler_spaces_until_thresh = thresh_in_dots - dots_below_thresh
+    out_str = '.'*dots_below_thresh + ' '*filler_spaces_until_thresh + '|'
+    if dots_total > thresh_in_dots:
+      dots_above_thresh = dots_total - thresh_in_dots
+      out_str += '*'*dots_above_thresh
+    total_secs = int(latency[0])
+    time_h = (int)(total_secs / 60)
```

Comment:
You can also import division from future (`from __future__ import division`) and use `//` for an integer division.